recipes/libuv/all/patches/1.40.0/fix-cmake.patch

```diff
@@ -0,0 +1,75 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e648b00..080f403 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -326,13 +326,17 @@ if(APPLE OR CMAKE_SYSTEM_NAME MATCHES "DragonFly|FreeBSD|Linux|NetBSD|OpenBSD")
+   list(APPEND uv_test_libraries util)
+ endif()
+ 
+-add_library(uv SHARED ${uv_sources})
+-target_compile_definitions(uv
+-  INTERFACE
+-    USING_UV_SHARED=1
+-  PRIVATE
+-    BUILDING_UV_SHARED=1
+-    ${uv_defines})
++add_library(uv ${uv_sources})
```

Comment:
I know that patches of previous versions in CCI are the same than this one, but maybe we should set `OUTPUT_NAME` to `uv_a` if static to be consistent with upstream names. Could be done in another PR.